test/widgets/emoji_reaction_test.dart

```diff
@@ -556,6 +556,32 @@ void main() {
         debugNetworkImageHttpClientProvider = null;
       });
 
+      testWidgets('text emoji list items must have correct height', (tester) async {
+        // Regression test for #1587
+        tester.platformDispatcher.textScaleFactorTestValue = 2.0;
+        addTearDown(tester.platformDispatcher.clearTextScaleFactorTestValue);
+
+        final message = eg.streamMessage();
+        // Set up the emoji picker, which leaves it open.
+        await setupEmojiPicker(tester, message: message, narrow: TopicNarrow.ofMessage(message));
```

Comment:
nit: Remove the comment; it describes something that's already obvious from reading the code